bridges/IvooxBridge.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * IvooxRssBridge
+ * Returns the latest search result
+ * TODO: support podcast episodes list
+ */
+class IvooxBridge extends BridgeAbstract
+{
+	const NAME = 'Ivoox Bridge';
+	const URI = 'https://www.ivoox.com/';
+	const CACHE_TIMEOUT = 10800; // 3h
+	const DESCRIPTION = 'Returns the 10 newest episodes by keyword search';
+	const MAINTAINER = 'xurxof'; // based on YoutubeBridge by mitsukarenai
+	const PARAMETERS = array(
+		'Search result' => array(
+			's' => array(
+				'name' => 'keyword',
+				'exampleValue' => 'test'
+			)
+		)
+	);
+	private function ivGetSimpleHTMLDOM($url)
+	{
+		return getSimpleHTMLDOM(
+		$url, 
```

Comment:
No whitespace at end of line.
Same line 25, 26, 38, 29, 30, 31, 32, 33, 34, 60, 63, 67, 96, 109